activities/communication/communications-plan.md

```diff
@@ -25,102 +25,198 @@ We believe it’s more important to talk concretely about what we’ve already d
+* think tanks, for example FSFE, Ford Foundation, New America Foundation,  Beeck Center for Social Impact and Innovation or Code for America
+* potential codebase communities and stakeholders - including commercial vendors
 
-We believe [these are potential contributors to the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).
+## Success criteria and indicators for target fora
 
-### Influencers
+Nota bene: none of these goals contain 'hype' or 'coolness' because being perceived as trendy or zeitgeisty /won't/ help us build the trustworthy reputation or longterm relationships we need to succed.
 
-These are domain celebrities (both individuals and organizations) in open source, government digital transformation and civic tech.
+### Public tech fora
 
-## Unique selling proposition
+Our goal for engagement with this group is:
 
-Though many specific things we do are also offered by other organizations, we're the only organization focused on open source software for public organizations that does all of this together:
+* they're aware of us and our mission, and believe this is the right thing to do
+* they engage critically with our content, use it and strengthen it
+* they talk about us with other people
 
-* is non-profit and member owned with membership only open to public organizations
-* offers codebase stewardship for different projects
-* offers productization and marketing of the codebases
-* actually provides guidance and support to help public organizations make their code open
-* isn’t tied to the EU project lifecycle
-* has a reproducible process suitable not just for cities, but all public organizations
-* is international, not national
+Success is shown these responses:
 
-We are also actively building our ecosystem by:
+* being asked to join a conversation
+* Github issues/contributions to our tools
+* recommendations or referrals
+* joining our events
+* endorsing/hyping our events/co*hosting our events
+* invitations to speak at/contribute to their platforms (publications and events)
 
-* connecting members with like-minded parties (members and other public organizations, affiliates and structural funders)
-* boosting awareness of public code
-* setting standards and creating reusable resources (by providing thought leadership!)
-* demonstrating the value of public code with well documented, public business cases and risk analyses
-* nurturing our community online, and – where sensible – in person
+### Public organisation leadership
+
+Our goal for engagement with this group is:
+
+* they're aware of us and our mission, and believe this is the right thing to do
+* they believe we can add value to their organizations
+* we're not an organizational risk to them
+* they think public code is achievable in their organization
+* they're willing to invest political capital because they think will add to their careers (good optics?)
+
+Success is shown these responses:
+
```

Comment:
Should we add: Request to collaborate (on projects through codebase stewardship) and Request to join as members

Reply:
done.